.gitignore

```diff
@@ -15,6 +15,7 @@ pids
 libs/
 node_modules/
 bower_components/
+*.js.map
```

Comment:
On the IDE? Probably yes, but I can't think of a good reason to accept js.map into the SCM. I can send a new PR for removing it from there, though.